frontend/src/views/cluster/Nodes/NodeDiagnosticWarnings.vue

```diff
@@ -0,0 +1,49 @@
+<!--
+Copyright (c) 2024 Sidero Labs, Inc.
+
+Use of this software is governed by the Business Source License
+included in the LICENSE file.
+-->
+<template>
+  <div class="flex flex-col flex-1">
+    <t-list-item v-for="diagnostic in diagnostics" :key="diagnostic.id" disable-border-on-expand>
+      <a class="diagnostic-item" :href="'https://talos.dev/diagnostic/' + diagnostic.id" target="_blank">
+        <t-icon icon="warning" class="w-4 h-4 text-yellow-400 mr-2"/>
+        <div class="text-yellow-400">{{ diagnostic.message }}</div>
+      </a>
+      <template #details>
+        <div class="diagnostic-sublist">
+          <div class="diagnostic-subitem" v-for="(detail, index) in diagnostic.details" :key="index">
+            <t-icon icon="dot" class="w-4 h-4 mr-2"/>
+            <span>{{ detail }}</span>
```

Comment:
probably adding `class="flex-1"` here will fix the icon changing size bug

Reply:
yep, done that, fixed the issue, thanks